main.py

```diff
@@ -129,6 +130,7 @@ def perform_TAS():
             # Blitzball testing logic
             if game.state == "Luca" and game.step == 3:
                 area.dream_zan.new_game(game.state)
+                import load_game
```

Comment:
Is there a reason we are doing these inline rather than at the top of the file?

Reply:
No good one as far as I know. I was referring to how the older code did it.